tests/kubectl-cmd.sh

```diff
@@ -5,4 +5,4 @@ set -xe
 export end_point='http://'$1':9090/'$3
 echo $2
 #kubectl run --quiet curl-isaac --image=tutum/curl -i --tty --restart=Never --rm -n sm -- curl --request POST --data '{ "data": "foo" }' $end_point
```

Comment:
please remove the comment

Reply:
This comment should stay. It is an example of what need to be passed

Reply:
👍  , It'll be nice if we make the comment explanatory instead just throwing it in there.

Reply:
ok